### PR TITLE
Restoring larger images

### PR DESCRIPTION
**Pull request by tfoote**
This is restoring the larger images on the robot overview pages but keeps the special pages w/o images such as the about and config pages the same. 

This will restore the visual impact of the images and help make the site more interesting to browse for non-technical audiences.

image

image

image

### CODE REVIEW

File: _layouts/minimal.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <body>
         {% include svg-icons.html %}
         {% include header-post.html %}
-        <section class="post">
+        <section class="minimal-post">
```
Comment (wjwwood) [inline _layouts/minimal.html:11]: This change (I think) as well as the increased page image size are (in my opinion) negatively impacting the appearance, before:

screen shot 2017-04-17 at 1 30 45 am

After:

screen shot 2017-04-17 at 1 30 26 am

The black on white icon looks out of place when other items are in purple (like the hamburger icon and search icon). I think the banner bar makes that look better. Also, the size of the icon makes start to look jagged on my screen (higher dpi screens in general). I think the non-white background also helps to hide the aliasing of the curves in the icon shape.

I don't know why you changed this particular item, but I think it looked better before.

File: contributing.md [hunk 2]
```diff
@@ -13,6 +12,8 @@ categories:
 twitter_text: 'How to use this template'
 introduction: 'How to use this template'
 ---
+<img itemprop="image" class="img" style="width: 200px; padding: 30px" src="/assets/img/contribute.png" alt="ROS nine dots">
```
Comment (wjwwood) [inline contributing.md:15]: Ok, I see what you're trying to do, but I'd recommend you try and replicate the previous style by including the banner div.

File: _layouts/minimal.html (hunk 1 above)
Comment (wjwwood) [reply _layouts/minimal.html:11]: I also think you need to keep in mind that the image on the robot pages really big makes the information that ends up on smaller screens limited, for example this it the turtlebot page on my screen:

screen shot 2017-04-17 at 1 40 18 am

**Comment by tfoote** [comment]
Closing in favor of #30

**Pull request closed without merging**